main.go

```diff
@@ -72,7 +72,14 @@ var defaultWatchRepositoriesPipeline = []string{
 	"mender-connect",
 }
 
-var githubRepoToGitLabProject = map[string]string{
+// Mapping https://github.com/<org> -> https://gitlab.com/Northern.tech/<group>
+var gitHubOrganizationToGitLabGroup = map[string]string{
+	"mendersoftware": "Mender",
+	"cfengine":       "CFEngine",
+}
+
+// Mapping of special repos that have a custom group/project
+var gitHubRepoToGitLabProject = map[string]string{
```

Comment:
Suggestion:
`githubRepoToGitlabProjectSpecial` or something similar

Reply:
I noticed here and earlier that you use "github" or "Gitlab" instead of "gitHub" or "GitLab". Do you have an strong opinion on this?

wrt the suggestion itself: okey.

Reply:
No, not at all. I just haven't realized till now that they actually write `GitHub` and `GitLab`